.github/workflows/pre-release-continuous-test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Run continuous pre release testing
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - nightly
+  schedule:
+    # Run hourly
+    - cron: "0 * * * *"
```

Comment:
@perryjrandall - I think running this hourly might be an overkill and also doesn't allow 2-3 hours performance benchmark to run. What are the thoughts around making this every 6 hours lets say?